bootloader/src/pyi_archive.c

```diff
@@ -333,6 +333,9 @@ int pyi_arch_open(ARCHIVE_STATUS *status)
 #elif defined(__APPLE__)
     int i;
     int alignment = 4096;
+#else
```

Comment:
Yeah I was thinking I could actually work out the best offset to use late but I'll just leave a note.
t